aws_sagemaker_studio/sagemaker_studio_image_build/xgboost_bring_your_own/Batch_Transform_BYO_XGB.ipynb

```diff
@@ -0,0 +1,1112 @@
+    "        },\n",
+    "        {\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Action\": [\n",
+    "                \"ecr:CreateRepository\",\n",
+    "                \"ecr:BatchGetImage\",\n",
+    "                \"ecr:CompleteLayerUpload\",\n",
+    "                \"ecr:DescribeImages\",\n",
+    "                \"ecr:DescribeRepositories\",\n",
+    "                \"ecr:UploadLayerPart\",\n",
+    "                \"ecr:ListImages\",\n",
+    "                \"ecr:InitiateLayerUpload\",\n",
+    "                \"ecr:BatchCheckLayerAvailability\",\n",
+    "                \"ecr:PutImage\"\n",
+    "            ],\n",
+    "            \"Resource\": \"arn:aws:ecr:*:*:repository/sagemaker-studio*\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Action\": \"ecr:GetAuthorizationToken\",\n",
+    "            \"Resource\": \"*\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Action\": [\n",
+    "              \"s3:GetObject\",\n",
+    "              \"s3:DeleteObject\",\n",
+    "              \"s3:PutObject\"\n",
+    "              ],\n",
+    "            \"Resource\": \"arn:aws:s3:::sagemaker-*/*\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Action\": [\n",
+    "                \"s3:CreateBucket\"\n",
+    "            ],\n",
+    "            \"Resource\": \"arn:aws:s3:::sagemaker*\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Action\": [\n",
+    "                \"iam:GetRole\",\n",
+    "                \"iam:ListRoles\"\n",
+    "            ],\n",
+    "            \"Resource\": \"*\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Action\": \"iam:PassRole\",\n",
+    "            \"Resource\": \"arn:aws:iam::*:role/*\",\n",
+    "            \"Condition\": {\n",
+    "                \"StringLikeIfExists\": {\n",
+    "                    \"iam:PassedToService\": \"codebuild.amazonaws.com\"\n",
+    "                }\n",
+    "            }\n",
+    "        }\n",
+    "    ]\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Let's inspect the role we have created for our notebook here:\n",
+    "import boto3\n",
+    "import sagemaker\n",
+    "from sagemaker import get_execution_role\n",
+    "\n",
+    "role = get_execution_role()\n",
+    "sess = sagemaker.Session()\n",
+    "region = boto3.session.Session().region_name\n",
+    "print(\"Region = {}\".format(region))\n",
+    "sm = boto3.Session().client('sagemaker')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Check the SageMaker SDK Version. \n",
```

Comment:
just put this in the setup notebook with :
```
!{sys.executable} -m pip install "sagemaker>=1.71.0,<2.0.0"
```

Reply:
And in the setup notebook's prerequisites section, clearly state that these notebooks expect SM PySDK v1.71, and are not compatible w/ v2.
Though... if we're adding new notebooks.... why wouldn't we make it v2?

Reply:
Aaron, unfortunately the studio build CLI doesn't work with the SDK version 2. We tried it. Since this is the only set up we need and its just 3 lines of code, i've left it in this notebook for now.

Reply:
Given that these notebooks may be used for different use cases; it may make sense for them to be self-sufficient.

Reply:
added a line clearly stating that notebooks expect a specific version of SM SDK.